frontend/webpack.config.ts

```diff
@@ -326,12 +326,14 @@ const config: Configuration = {
     new CopyWebpackPlugin([{ from: './packages/rhoas-plugin/locales', to: 'locales' }]),
     new CopyWebpackPlugin([{ from: './packages/git-service/locales', to: 'locales' }]),
     new CopyWebpackPlugin([{ from: './packages/gitops-plugin/locales', to: 'locales' }]),
-    new CopyWebpackPlugin([{ from: './packages/kubevirt-plugin/locales', to: 'locales' }]),
     new CopyWebpackPlugin([{ from: './packages/metal3-plugin/locales', to: 'locales' }]),
     new CopyWebpackPlugin([{ from: './packages/vsphere-plugin/locales', to: 'locales' }]),
     new CopyWebpackPlugin([
       { from: './packages/network-attachment-definition-plugin/locales', to: 'locales' },
     ]),
+    new CopyWebpackPlugin([
+      { from: './packages/network-attachment-definition-plugin/locales', to: 'locales' },
+    ]),
```

Comment:
are these lines duplicate?

Reply:
Yes, they were. I removed the duplicate. Thanks for catching that!